pkg/crawler/crawler.go

```diff
@@ -262,3 +308,204 @@ func (c *Crawler) fetchSHA1(url string) ([]byte, error) {
 	}
 	return sha1b, nil
 }
+
+func (c *Crawler) fetchAndSavePOMLicenseKeys(url string) ([]string, error) {
+	var keys []string
+	resp, err := c.http.Get(url)
+	if resp.StatusCode == http.StatusNotFound {
+		return keys, nil
+	}
+	if err != nil {
+		return keys, xerrors.Errorf("can't get pom xml from %s: %w", url, err)
+	}
+	defer resp.Body.Close()
+
+	var pomProject PomProject
+
+	decoder := xml.NewDecoder(resp.Body)
+	decoder.CharsetReader = charset.NewReaderLabel
+	err = decoder.Decode(&pomProject)
+
+	if err != nil {
+		return keys, xerrors.Errorf("can't parse pom xml from %s: %w", url, err)
+	}
+
+	if len(pomProject.Licenses) == 0 {
+		return keys, nil
+	}
+
+	for _, l := range pomProject.Licenses {
+		l.LicenseKey = getLicenseKey(l)
+
+		// update uniqueLicenseKeys map
+		c.uniqueLicenseKeys.Set(l.LicenseKey, l)
+
+		keys = append(keys, l.LicenseKey)
+	}
+
+	return keys, nil
+
+}
+
+func (c *Crawler) classifyLicense() error {
+	normalizedLicenseMap := make(map[string]string)
+
+	// prepare classifier data i.e create temporary files with license text to be used for classification
+	filesLicenseMap := c.prepareClassifierData()
+
+	if len(filesLicenseMap.Keys()) == 0 {
+		return nil
+	}
+
+	// classify licenses
+
+	// 1 minute is the timeout for license classification of a file
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	defer cancel()
+
+	// 1000 is the number of concurrent tasks spawned to process license files
+	errs := c.classifier.ClassifyLicensesWithContext(ctx, 1000, filesLicenseMap.Keys(), true)
```

Comment:
I think we can use number of parallel tasks from Option.

Reply:
Can we also disable classifier logger?
We can show  total number of licenses found or something like that.

Reply:
Fixed. 

> Can we also disable classifier logger?
Don't think thats possible. Don't see a way to override the logging configuration of the library